.github/ISSUE_TEMPLATE/dev-ticket.md

```diff
@@ -0,0 +1,33 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Story
+<!--- Describe the benefits of this change, ideally in the following format: -->
+<!--- As a < type of user >, I want < some goal > so that < some reason >. -->
+
+## Rationale
+<!--- Provide the relevant background information to udnerstand why this change is needed -->
+<!--- This is where you can provide more details thathelp the reader understand why you want to make this change -->
+
+## Implementation details
+<!--- Explain the steps you will take to implement this change -->
+
+## Priority
+<!--- How urgent is this issue? -->
```

Comment:
How urgent do you think this issue is and why?

Reply:
```suggestion
<!--- How urgent is this issue, and why? -->
```